docs/units-and-global-variables.rst

```diff
@@ -268,14 +268,21 @@ Members of Address Types
 ``<address payable>.send(uint256 amount) returns (bool)``
     send given amount of Wei to :ref:`address`, returns ``false`` on failure, forwards 2300 gas stipend, not adjustable
 
+.. warning::
+    ``send()`` and ``transfer()`` are deprecated and scheduled for removal.
+    See the section on :ref:`address` for more information.
```

Comment:
Please also add a link to "Members of Addresses" at the beginning of the whole section.

Reply:
Also would not hurt to have the warnings link specifically to the part on `send()` and `transfer()` rather than the whole section about the `address` type, since it's long.